### PR TITLE
Overall challenge document review

### PR DESCRIPTION
**Pull request by ben47955**

### CODE REVIEW

File: SPL-Challenges.tex [hunk 1]
```diff
@@ -110,21 +110,12 @@
 \setcounter{page}{1}
 
 \section{Introduction}
-At RoboCup \RCYear, the Standard Platform League will hold three technical challenges, which are described in this document.
-RoboCup \RCYear awards a trophy for winning the overall ranking of the three challenges and the option for pre-qualification if a team is not pre-qualified by other means.
+At RoboCup \RCYear, the Standard Platform League will hold one technical challenge, which are described in this document.
```
Comment (arnemoos) [inline SPL-Challenges.tex:113]: > which is described

Comment (ben47955) [reply SPL-Challenges.tex]: Updated

**Comment by arnemoos** [comment]
Since you removed all the includes, it may happen that after this merge the shared autonomy challenge gets merge conflicts.

**Review by arnemoos — APPROVED** [review]

**Comment by ben47955** [comment]
Resolved the conflict. Nothing should have change. 
Also reviewed the whole document with the challenge included.

**Pull request merged**